### ISSUE TITLE
Vulkan works - dxvk doesn't (empty d3d11.log and dxgi.log files)

### ISSUE THREAD

**Issue by FiraWolf** [doitsujin/dxvk#190]
Hey,

I've been trying to get dxvk running for some hours now but to no avail.
I can confirm that Vulkan via Wine does work as intended (cube.exe works, etc).
When trying to run any game with dxvk via the following command

`DXVK_HUD=1 DXVK_LOG_LEVEL=debug DXVK_DEBUG_LAYERS=1 WINEPREFIX=/home/user/.wineCustom/dxvk64/ WINEDEBUG=-all,+vulkan wine64 /path/to/game.exe &> output.txt
`
the game will just start with usual DX11. The game_d3d11.log and game_dxgi.log files are just empty and output.txt is empty as well (since no vulkan messages get logged).

I'm kinda sure that my setup is right, since I tried it manually a few times and via setup scripts. I also tried different Wine versions, different Windows Vulkan SDK versions and a 32bit prefix.

Does someone know how to tackle this problem? Since I can't even get debug messages, I don't know how to proceed..

### System information
- GPU: NVidia GTX 970
- Driver: 390.42-3
- Wine version: **3.4-staging-pba**, 3.4-staging, 3.4
- DXVK version: 0.31-4
- VulkanSDK version: **1.0.51.0**, 1.0.68.0, 1.1.70.0
(bold = my main versions)

### Log files
- $WINEPREFIX/drive_c/Program\ Files\ \(x86\)/VulkanRT/1.0.51.0/vulkaninfo.exe: 
[vulkaninfo.txt](https://github.com/doitsujin/dxvk/files/1835507/vulkaninfo.txt)

**Comment by doitsujin** [comment]
Which game are you even trying to run?

**Comment by FiraWolf** [comment]
Guild Wars 2, Battlerite and Minion Masters were used for testing but mainly GW2, since it doesn't require Steam

**Comment by Guy1524** [comment]
Make sure that both 32 bit and 64 bit vulkan applications run btw

**Comment by steemandlinux** [comment]
GW2 doesn`t support d3d 10/11

**Comment by doitsujin** [comment]
In general, if the log files are *empty*, the game loads the D3D11 and DXGI DLLs, but doesn't *use* them at all. Make sure to run your games in Dx11 mode, if available.

**Comment by FiraWolf** [comment]
@Guy1524  Yes, they do.
@steemandlinux  Damn, I could have sworn it uses dx11. No wonder why dxvk doesn't work. I'm certain the other two do use dx11, though.
@doitsujin So **some** applications load these DLLs even if they don't use them? That's definitely not the case for all directx applications. Right now I tested WoW-3.3.5a and it was running 100% d3d9 without generated log files from dxvk. 

Anyways, I had the GOG copy of Witcher 3 laying around and - in fact - it works as intended. Seems like I just tested the wrong game(s)..

Some more testing: It seems like just UNITY games are affected. I can confirm that the two games in question (see above, BR and MM) start in d3d11 mode but don't generate an output in their log files. On the other hand FFXV (on the same prefix and steam.exe) does. If anyone can confirm a similar behavior, let me know and we should open a separate issue for this.

Nonetheless, I'll close this issue. **Thanks for the help!** I Hope the information can help others as well.
